sensor_msgs/PointCloud2 cloudMap
sensor_msgs/PointCloud2 cloudObstacles
